src/common/dataHooks/useSites.js

```diff
@@ -0,0 +1,24 @@
+import { useQuery } from 'react-query'
+
+// FPCC
+import { SITES } from 'common/constants'
+import api from 'services/api'
+import placeholder from 'images/cover-thumbnail.png'
+
+export default function useSites() {
+  const response = useQuery([SITES], () => api.site.get())
+  const formattedData = response?.data?.map((language) => ({
+    language: language?.language,
+    sites: language?.sites.map((site) => ({
+      uid: site?.id,
+      title: site?.title,
+      sitename: site?.slug,
+      visibility: site?.visibility,
```

Comment:
I'm looking at this and realizing we should check whether the visibility keywords need any tweaking. Are the locks still working on the Explore Languages page for private sites? If not we might need to adjust this a bit.